src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/components/AnalysesQueue.java

```diff
@@ -5,6 +5,7 @@
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
+
```

Comment:
Don't think this is supposed to be here

Reply:
Removed in cccb6498fc91d292d20be2ab190d78f077010016 :)